<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>971</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_main" stretch="1,1,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_l">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_putinfile">
        <property name="title">
         <string>输入图像目录：</string>
        </property>
        <widget class="QLineEdit" name="lineInPath">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>391</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_inchoose">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>40</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_ratio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>裁剪比例：</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>461</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_ratio">
          <item>
           <widget class="QRadioButton" name="fiveinches">
            <property name="text">
             <string>5寸（10:7）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sixinches">
            <property name="text">
             <string>6寸（3:2）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sixinchesLong">
            <property name="text">
             <string>大6寸（4:3）</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_direction">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>裁剪方向：</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>461</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_direction">
          <item>
           <widget class="QRadioButton" name="originalD">
            <property name="text">
             <string>原本方向</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="verticalD">
            <property name="text">
             <string>横向</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="horizonalD">
            <property name="text">
             <string>竖向</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_mode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>裁剪方式</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>311</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_mode">
          <item>
           <widget class="QRadioButton" name="smartCropping">
            <property name="text">
             <string>智能裁剪</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="leaveBlank">
            <property name="text">
             <string>四周留白</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_putout">
        <property name="title">
         <string>输出图像目录：</string>
        </property>
        <widget class="QLineEdit" name="lineOutPath">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>391</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_outchoose">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>40</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_r">
      <item>
       <widget class="QLabel" name="logTitle">
        <property name="text">
         <string>输出面板：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="logText">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_progress">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="plabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openfile">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>打开文件夹</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
